lib/grape-swagger.rb

```diff
@@ -111,6 +119,33 @@ def combine_namespace_routes(namespaces)
         end
       end
 
+      def sub_routes_from(parent_route, sub_namespaces)
+        sub_ns_paths = sub_namespaces.collect { |ns_name, _| ["/#{ns_name}", "/:version/#{ns_name}"] }
+        sub_routes = parent_route.reject do |route|
```

Comment:
`sub_routes` is not needed here, this could just be left out to return the result of `reject`.